--- origsrc/kQOAuth/src/src.pro	2013-12-25 14:11:16.000000000 -0600
+++ src/kQOAuth/src/src.pro	2013-12-25 14:18:59.525922700 -0600
@@ -89,11 +89,18 @@ else:unix {
 }
 else:windows {
     INSTALL_PREFIX = $$[QT_INSTALL_PREFIX]
+
+    # this creates a pkgconfig file
+    system( ./pcfile.sh $${INSTALL_PREFIX} $${VERSION} )
+    pkgconfig.files = kqoauth.pc
+
     target.path = $${INSTALL_PREFIX}/lib
     headers.path = $${INSTALL_PREFIX}/include/QtKOAuth
+    pkgconfig.path = $${target.path}/pkgconfig
     INSTALLS += \
         target \
         headers \
+        pkgconfig \
         features
 }
 
